tests/test_comparison.py

```diff
@@ -95,7 +95,7 @@ def circuit(measurement):
 
         lightning(qml.expval(qml.PauliZ(0)))
         # pylint: disable=protected-access
-        lightning_state = dev_l._statevector.state if dev_l._new_API else dev_l.state
+        lightning_state = dev_l._statevector.state
```

Comment:
That's OK to have it, but we can probably call `dev_l._statevector.state` directly on the np.close function.